chain/messagepool/messagepool.go

```diff
@@ -55,6 +55,7 @@ var baseFeeLowerBoundFactor = types.NewInt(10)
 var baseFeeLowerBoundFactorConservative = types.NewInt(100)
 
 var MaxActorPendingMessages = 1000
+var MaxUntrustedActorPendingMessages = 10
```

Comment:
* Do we use that for messages coming from the messages pubsub?
* If yes, that's likely way too low, many bigger miners have many more messages that that per tipset

Reply:
no, we only use it for the gateway -- messages coming from pubsub use `Add`.